tests/sync/util.js

```diff
@@ -146,6 +146,7 @@ const syncFromDisk = require('../../sync/syncFromDisk');
  * @property {boolean} [studentGroupCreate]
  * @property {boolean} [studentGroupJoin]
  * @property {boolean} [studentGroupLeave]
+ * @property {string} helpLink
```

Comment:
Square brackets indicate an optional property:

```suggestion
 * @property {string} [helpLink]
```